internal/core/item_rendering.rs

```diff
@@ -451,6 +451,10 @@ pub trait ItemRenderer {
 
     fn draw_image_direct(&mut self, image: crate::graphics::Image);
 
+    fn draw_rect(&mut self, _size: LogicalSize, _brush: Brush) {
```

Comment:
Documentation would be nice.

I'm not seeing where this is used.

Reply:
I'll add a comment. It's used for the gradient window fill.